shared/middleware/files_api.rb

```diff
@@ -58,7 +58,7 @@ def file_too_large(quota_type)
   end
 
   def quota_crossed_half_used?(app_size, body_length)
-    app_size < FilesApi::max_app_size / 2 && app_size + body_length >= FilesApi::max_app_size / 2
+    app_size < max_app_size / 2 && app_size + body_length >= max_app_size / 2
```

Comment:
Nit for a followup PR:  Please parenthesize the subexpressions around && to make the precedence obvious